### ISSUE TITLE
[Feature Request]: Small UI redesign proposal

### ISSUE THREAD

**Issue by pkrasicki** [FreeTubeApp/FreeTube#3918]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

I would like to propose some small layout changes and background color changes for the default dark theme.

## Before
Trending:
![before](https://github.com/FreeTubeApp/FreeTube/assets/49721209/f9332fc3-e3f7-4c55-822d-9fa3588058fc)

## After
Trending:
![trending](https://github.com/FreeTubeApp/FreeTube/assets/49721209/c5f52d07-51fb-4087-8d9f-fe3cc5d7dcac)

Channel:
![channel](https://github.com/FreeTubeApp/FreeTube/assets/49721209/d2adf715-0a1b-4565-b600-b1cc15f5a6bf)

Video:
![video](https://github.com/FreeTubeApp/FreeTube/assets/49721209/0662c4a4-ce20-45f4-9cd6-7be6b42b603c)

Mobile:
![mobile](https://github.com/FreeTubeApp/FreeTube/assets/49721209/b113ba0e-3ab9-49b1-a3d6-4fd32f50ec51)

## Changes:
- new colors for the default dark theme:
main background: `#0f0f0f`
sideNav, topNav, card: `rgb(28,28,28)`
- removed card background color from most pages
- smaller paddings - less wasted space on desktop and mobile, on 1080p resolution 1 more video fits in each row
- page title is centered
- inactive tab text color is a little brighter
- padding-bottom for tabs is set to 10px instead of 15px

### Alternatives Considered

None.

### Issue Labels

visual improvement

### Additional Information
I can create a pull request implementing those changes.

_No response_

**Comment by absidue** [comment]
Personally I'm against all of the changes in this pull request, apart from the inactive tab colour. Let me explain.

The colour changes make the dark theme look a lot closer to the existing black theme, defeating the purpose of having multiple distinct themes.

While I see that your goal was to display as much content as possible, I think this proposed solution sacrifices usability, accessibility and looks, resulting in a cramped, overcrowded UI, that will overwhelm not just users with accessibility needs, but also everyone that isn't a productivity nerd. The "wasted" space as you call it, prevents that overcrowding.

Personally I think this pull request would worsen the user experience for most users.

**Comment by pkrasicki** [comment]
The goal wasn't really to fit as much stuff as possible, but after tabs were added to the subscriptions page and they take up a separate row, I noticed how much unused space there is in this application. For me it was getting in the way of viewing the content. Maybe it's because I've been using Piped lately (https://piped.video). My changes are inspired by that application. I understand if you don't like that style though.

The extra backgrounds also felt unnecessary and a bit distracting, so I removed them. The colors can be changed and you are right that it looks much closer to the black theme now.

I ended up doing those changes for videos and playlists too:
![video2](https://github.com/FreeTubeApp/FreeTube/assets/49721209/f028f7f3-4c27-4b1d-b589-a18525f3c3c3)

![video3](https://github.com/FreeTubeApp/FreeTube/assets/49721209/d4099767-dfdb-4f56-b8d4-27bac45d062b)

![playlist](https://github.com/FreeTubeApp/FreeTube/assets/49721209/f284a7bc-b40b-4f12-bb97-acb74248324f)

**Comment by pkrasicki** [comment]
@o-data I have uploaded all of my changes into branch https://github.com/pkrasicki/FreeTube/tree/ui-redesign. It's based on the latest 0.19 release, so if you don't mind being stuck with that version, you can compile FreeTube from my branch. If you haven't done that yet, maybe you will be able to find someone who will help you.

**Comment by absidue** [comment]
At the moment it is probably better leaving it as it is. Considering that we've been making changes to improve accessibility in FreeTube recently, it would be counter productive to change to a design and layout to something that makes it less accessible in other ways.

Unfortunately your goal is to display more content, but to get something that looks good and isn't overcrowded and overwhelming, you would probably have to make enough changes to the display of the list items, that you would end up displaying the same amount of content just with a different design.

Basically if you want a full width layout for FreeTube you would have to make displaying more content a much lower priority than it currently is for you.

**Comment by pkrasicki** [comment]
My goal isn't to display more information. It just happened as a result of removing the padding. It was the easiest for me to do nothing about it, but maybe I should have made the videos bigger. We can do that.

As a user I want to see the content. Everything else should take as little space as possible. You can see it in the design of other similar applications:

YouTube
![youtube](https://github.com/FreeTubeApp/FreeTube/assets/49721209/3d77ba30-629a-4352-a415-3eb20431c16c)

Piped
![piped](https://github.com/FreeTubeApp/FreeTube/assets/49721209/469c64f9-5437-467e-a1e0-10f3810979d2)

Odysee
![odysee](https://github.com/FreeTubeApp/FreeTube/assets/49721209/a07f203a-f52f-442f-b1ec-d4644e280af9)

They don't have a padding that takes up around 30% of screen width. They don't have an extra background around the content. Those are unnecessary things that we can easily remove and we can do it without negatively impacting accessibility. That is the main goal of my proposal.

**Comment by jasonhenriquez** [comment]
Hello @pkrasicki, after discussing with the FreeTube team, we do think this bundle of changes is too specific to warrant its own mode at this time, but we would most certainly be open to having a full-width option for the app. If you could create a PR that implements just the full-width change and adds it to the Theme Settings, we would be glad to have that as a feature. Our only other stipulation is that we would like it to have a default value of disabled before we consult the community. Would you be amenable to this suggestion?

**Comment by pkrasicki** [comment]
Hi @jasonhenriquez, sure, I will give it a try. It will look like this when enabled:

![full-width-only](https://github.com/FreeTubeApp/FreeTube/assets/49721209/76ebc3f6-729e-40d7-ba6e-bb2c74003f59)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @pkrasicki just checking in how is your PR coming along?

**Comment by pkrasicki** [comment]
Hi @efb4f5ff-1298-471a-8973-3d47447115dc, I'm busy at the moment and don't have the time to work on it this month, but the feature is mostly finished. I've put the changes into a new branch. Unfortunately I based the branch on master instead of development. It also needs to be rebased on top of the latest changes and there are merge conflicts. Other than that it should be done, but might require some more testing on mobile. I will try to get back to it next month.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Based on https://github.com/FreeTubeApp/FreeTube/pull/4379#issuecomment-1912542304 im closing this, as that was the only feature that interested us.
